scripts/run-pre-build-checks

```diff
@@ -5,12 +5,13 @@ set -e
 # `$-` yields the list of the options of the current shell; we'll only interested in whether `x` is present
 x_option=${-//[^x]/}
 
-./scripts/enforce-change-notes-updated-on-release-pr
+# pass -x option if this script is run with -x option itself
+bash ${x_option:+-x} ./scripts/enforce-change-notes-updated-on-pr-to-master
```

Comment:
Let's turn on shell debug in case of the scripts that execute completely differently on CI than locally